src/components/20-molecules/top-content-bar/story.js

```diff
@@ -57,13 +65,23 @@ export const TopContentBar = () => {
     }
 
     el.addEventListener('axa-top-content-bar-close', () => {
-      window.sessionStorage.setItem('top-content-bar-closed', 'true');
+      const ssVal = window.sessionStorage.getItem('top-content-bar-closed');
+      const ssArray = ssVal ? JSON.parse(ssVal) : [];
+      if (!ssArray.includes(el.dataset.configHash)) {
+        ssArray.push(el.dataset.configHash);
+      }
+      window.sessionStorage.setItem(
+        'top-content-bar-closed',
+        JSON.stringify(ssArray)
+      );
     });
   });
 
   const template = html`
     <h1>HEADER</h1>
     <axa-top-content-bar
+      style="display:none"
+      data-config-hash="123abc"
```

Comment:
does patterns lib have to be aware of the session-state logic? I know it's only demo code, but still I think everything related to this should be removed from here as it's purely the consuming application's logic.

Reply:
@Ben-Zahler You are right. The attribute data-config-hash is totally irrelevant for the inner component logic. I removed it.